src/vos/vos_space.c

```diff
@@ -38,15 +38,13 @@ frag_reserve_space(uuid_t uuid, daos_size_t *rsrvd, daos_size_t scm_tot)
 		return;
 	}
 
-	if (ovhd_sz > max_sz) {
+	if (ovhd_sz > max_sz)
 		ovhd_sz = max_sz;
-	}
 
-	if (ovhd_sz < min_sz && scm_tot >= 2 * (min_sz + rsrvd[DAOS_MEDIA_SCM])) {
+	if (ovhd_sz < min_sz && scm_tot >= 2 * (min_sz + rsrvd[DAOS_MEDIA_SCM]))
```

Comment:
what's the significance of "2 *" here, is that just an arbitrary estimation that is comparing to double the value?